.vib/codeigniter/goss/vars.yaml

```diff
@@ -0,0 +1,18 @@
+binaries:
+  - composer
+  - php-fpm
+  - gosu
+  - mysql
+files:
+  - paths:
+      - /opt/bitnami/codeigniter/app/Common.php
+      - /opt/bitnami/codeigniter/psalm.xml
+      - /opt/bitnami/codeigniter/composer.json
+directories:
+  - mode: "0775"
+    paths:
+      - /app
+root_dir: /opt/bitnami
+version:
+  bin_name: /opt/bitnami/codeigniter/spark
+  flag: --version
```

Comment:
Beware that the output this produces is:

```
...
CodeIgniter v4.3.1 Command Line Tool - Server Time: 2023-03-02 16:43:28 UTC+00:00
...
```

It might not be a problem, but it seems that version is related to the CLI itself, so there might be a mismatch between the product and the CLI.

Reply:
It is the only way to check for a version, I could remove the whole test if it's better.

Reply:
I would first investigate where are we getting both the CLI and product from. If they are both taken from the same source code, have different release cycles, etc. Unless absolutely necessary, we should maintain the test.

Reply:
@FraPazGal, the spark cli is included in the upstream tarball, it's a PHP script that runs codeigniter itself:

```
$app = Config\Services::codeigniter();
$app->initialize();
```
So I think we can be certain that the version will always coincide with the application's version.